cli/util/convert.go

```diff
@@ -57,16 +57,16 @@ func NewCommands() []*cli.Command {
 			Action:   cmdargs.EnsureNotEmpty("container"),
 		},
 		&cli.StringFlag{
-			Name:     "block-attribute",
-			Usage:    "Attribute key of the block object",
-			Required: true,
-			Action:   cmdargs.EnsureNotEmpty("block-attribute"),
+			Name:   "block-attribute",
+			Usage:  "Attribute key of the block object",
+			Value:  neofs.DefaultBlockAttribute,
```

Comment:
Please update the docs.

Reply:
We do this every time, hence I've refactored the docs.